broker-daemon/orderbook/index.js

```diff
@@ -337,6 +341,39 @@ class Orderbook {
       sequence
     }
   }
+
+  /**
+   * Updates retry count and calculates next delay period if the
+   * MAX_RETRY_INTERVAL hasn't been surpassed
+   *
+   * @private
+   * @return {void}
+   */
+  _updateExponentialDelayPeriod () {
+    if (this._reachedMaxInterval) {
+      return
+    }
+
+    this._retries++
+    const newDelay = (2 ** this._retries - 1) * 1000
+    if (newDelay > MAX_RETRY_INTERVAL) {
```

Comment:
why not do something like `this._delay = Math.min(newDelay, MAX_RETRY_INTERVAL)` ?